database/factories/ModelFactory.php

```diff
@@ -71,6 +68,23 @@
     ]);
 });
 
+/**
+ * Specifically an Admin Centre User [foodmatters_user]
+ */
+$factory->defineAs(App\CentreUser::class, 'FMUser',function (Faker\Generator $faker) use ($factory){
+    static $password;
+
+    return [
+        'name' => $faker->name,
+        'email' => $faker->unique()->safeEmail,
+        'password' => $password ?: $password = bcrypt('secret'),
+        'remember_token' => str_random(10),
+        'role' => 'foodmatters_user',
+        'downloader' => true
```

Comment:
Do you also have `foodmatters_users` who cannot download?

Reply:
At this point _no_ but there's a possible thought that we might in the future.